packages/sdk/odahuflow/sdk/clients/oidc.py

```diff
@@ -65,14 +65,18 @@ def fetch_configuration(self) -> None:
         Fetch OpenID Provider configuration making HTTP request
         :return:
         """
-        response = requests.get(self.configuration_url)
-        if response.status_code == 200:
-            self._config_json = response.json()
-        elif 400 <= response.status_code < 600:
-            raise RuntimeError(f'Some error during attempt to fetch OpenID Provider configuration. '
-                               f'Reason: {response.reason}')
-        else:
-            raise RuntimeError(f'Not expected status code: {response.status_code}')
+        try:
+            response = requests.get(self.configuration_url)
```

Comment:
I'd suggest to leave only this line in `try` block, the `if-elif-else` can go after catching the exception. The less nested code is - the better.

Reply:
OK, Agree. Fixed